src/plugins/console/public/application/hooks/use_send_current_request_to_es/use_send_current_request_to_es.ts

```diff
@@ -31,44 +31,24 @@ export const useSendCurrentRequestToES = () => {
     services: { history, settings, notifications, trackUiMetric },
   } = useServicesContext();
 
-  const { requestInFlight } = useRequestReadContext();
   const dispatch = useRequestActionContext();
 
   return useCallback(async () => {
-    if (requestInFlight) {
-      notifications.toasts.addWarning({
-        title: i18n.translate('console.requestAlreadyInFlightTitle', {
-          defaultMessage: 'Cannot send request right now.',
-        }),
-        toastLifeTimeMs: 3000,
-      });
-      return;
-    }
-    dispatch({ type: 'sendRequest', payload: undefined });
     try {
       const editor = registry.getInputEditor();
       const requests = await editor.getRequestsInRange();
       if (!requests.length) {
-        dispatch({
-          type: 'requestFail',
-          payload: {
-            response: {
-              value: 'No requests in range',
-              contentType: 'text/plain',
-              statusCode: 0,
-              statusText: 'None',
-              timeMs: 0,
-            },
-            request: {
-              data: null,
-              method: '',
-              path: '',
-            },
-          },
-        });
+        notifications.toasts.add(
+          i18n.translate('console.notification.noReqeustSelectedTitle', {
+            defaultMessage:
+              'It looks like your cursor is not on a request. Please select a request by placing your cursor on it.',
```

Comment:
Nit: I think our copywriting guide discourages phrases like "it looks like" and niceties like "please". My suggestion is: "No request selected. Select a request by placing the cursor inside it."